docs/preset-env.md

```diff
@@ -264,18 +264,85 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-This option configures how `@babel/preset-env` handles polyfills.
+This option configures how `@babel/preset-env` handles polyfills and whether optimizations are applied to only include polyfills that are needed.
 
-When either the `usage` or `entry` options are used, `@babel/preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
-
-Since `@babel/polyfill` was deprecated in 7.4.0, we recommend directly adding `core-js` and setting the version via the [`corejs`](#corejs) option.
+When specifying polyfilling behavior, you will want to include `core-js` and probably `regenerator-runtime` dependencies:
 
 ```sh
-npm install core-js@3 --save
+npm install --save core-js regenerator-runtime
+```
+
+You'll also need to set the core-js version via the [`corejs`](#corejs) option:
+
+```json
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "corejs": 3
+      }
+    ]
+  ]
+}
+```
 
-# or
+Generally, `"usage"` meets the needs of most users by only including the polyfills that are needed.
```

Comment:
This might be a tricky recommendation for users who have dependencies in `node_modules` that rely on certain built-ins being available that aren't run through Babel.

Reply:
I hadn't even considered that. Sounds like a nuance worth mentioning, I'll include that...